workshops/2017/react-chicago-july.html

```diff
@@ -107,26 +107,30 @@
         <div class="col-md-8">
           <div class="panel panel-default">
             <div class="panel-body large-text">
-              <p>Have you ever wondered how many times you planned to learn React and something else taken away your weekend?
-                Have you ever counted how many times you saw a tutorial or a blog and wished you could build an app with
-                React?
+              <p>
+                Have you planned on learning a new framework like React, but lack time or motivation, even on the weekends?
```

Comment:
Will this be.. 
Have you planned on learning a new framework like React, but felt lack of time or motivation, even on the weekends?